js/app/modules/card/list.js

```diff
@@ -54,6 +63,12 @@ const List = new Module.Class({
         if (this.model.duration) {
             this.set_duration_label(this._context_label, this.model.duration);
         }
+
+        if (this.show_context) {
+            this._context_widget = this.create_context_widget_from_model();
+            this._context_frame.add(this._context_widget);
```

Comment:
you are not showing the widget, so you rely on the API user to call show_all() for this to work